docs/book/intro.md

```diff
@@ -1,5 +1,8 @@
 # Introduction to zend-console
 
+## Deprecated
+Both the zend-console an the [zend-mvc-console](https://docs.zendframework.com/zend-mvc-console/) components will likely not be maintained long-term, as there are more complete implementations available elsewhere. We strongly urge developers to start migrating their console tooling to use other libraries, such as [symfony/console](https://github.com/symfony/console).
```

Comment:
Please check the spelling mistake in the first sentence. Thanks!

Reply:
Done. Thanks.